app/api/db/models/user_models.py

```diff
@@ -0,0 +1,75 @@
+from datetime import date
+from typing import Optional
+from uuid import UUID
+
+from sqlalchemy import Boolean, Column, Date, ForeignKey, Integer, Text
+from sqlalchemy.dialects.postgresql import UUID as postgresUUID
+from sqlalchemy.orm import Mapped, relationship, scoped_session
+
+import api.logging
+from api.db.models.lookup import LookupTable
+
+from .base import Base, TimestampMixin, uuid_gen
+
+logger = api.logging.get_logger(__name__)
+
+#################################
+# Lookup Tables
+#################################
+
+
+class LkRole(Base, TimestampMixin):
+    __tablename__ = "lk_role"
+    role_id: int = Column(Integer, primary_key=True, autoincrement=True)
+    role_description: str = Column(Text, nullable=False)
+
+    def __init__(self, role_id: int, role_description: str):
+        self.role_id = role_id
+        self.role_description = role_description
+
+
+class Role(LookupTable):
+    model = LkRole
+    column_names = ("role_id", "role_description")
+
+    USER = LkRole(1, "User")
+    ADMIN = LkRole(2, "Admin")
+    THIRD_PARTY = LkRole(3, "Third Party")
```

Comment:
i don't understand why there's a need for a `Role` and a `LkRole` type. could you help me understand that?